### PR TITLE
Draft pick

### PR DESCRIPTION
**Pull request by PolinaRomanchenko**
1) Added functionality
2) Added input error handling
3) Added description of exceptions

### CODE REVIEW

File: pypuck/pypuck.py [hunk 1]
```diff
@@ -105,6 +103,8 @@ def draft_pick(pick_number = 1, round_number=None, year=None):
     The function returns information about draft picks for the specified parameters and stores them in a pandas data frame.
     If year is not specified then all of the draft picks for that year will be returned. If no round is specified
```
Comment (chuusan) [inline pypuck/pypuck.py]:
```suggestion
    If year is not specified, then all of the draft picks for all year will be returned. If no round is specified 
```

File: pypuck/pypuck.py [hunk 2]
```diff
@@ -105,6 +103,8 @@ def draft_pick(pick_number = 1, round_number=None, year=None):
     The function returns information about draft picks for the specified parameters and stores them in a pandas data frame.
     If year is not specified then all of the draft picks for that year will be returned. If no round is specified 
     the dataframe will include all of the players with chosen pick number from every round.
```
Comment (chuusan) [inline pypuck/pypuck.py]:
```suggestion
    the data frame will include all players with chosen pick number from every round.
```

**Review by chuusan — CHANGES_REQUESTED** [review]
The code looks good for me. I just listed some writing  suggestions.

File: pypuck/pypuck.py (hunk 2 above)
Comment (PolinaRomanchenko) [reply pypuck/pypuck.py]: Thanks!

**Review by chuusan — APPROVED** [review]
All good!

**Pull request merged**